TestLib/EarlGreyImpl/GREYElementInteractionErrorHandler.m

```diff
@@ -78,7 +79,13 @@ void GREYHandleInteractionError(__strong GREYError *interactionError,
       id<GREYFailureHandler> failureHandler =
           [NSThread mainThread].threadDictionary[GREYFailureHandlerKey];
       // TODO(b/147072566): Will show up a (null) in rotation.
-      [failureHandler handleException:exception details:matcherDetails];
+      if (GREYShouldUseErrorFormatterForError(interactionError)) {
+        // Eventually this check will not be needed, and the error's description will
+        // always be passed as the `details`
+        [failureHandler handleException:exception details:interactionError.description];
+      } else {
```

Comment:
Since interactionError.description is being used directly you can move this to line 54.

     if (GREYShouldUseErrorFormatterForError(interactionError)) {
        [failureHandler handleException:exception details:interactionError.description];
      } else {
        NSMutableString *matcherDetails;
        ......
      }

Reply:
Resolved. It had to be more complex than just moving it to 54, so take a look